core/services/relay/evm/mercury/wsrpc/mocks/server.go

```diff
@@ -0,0 +1,76 @@
+package mocks
```

Comment:
Did this need to move out of the test package? It is surprising to see non-generated code in a mocks package. Do we do that anywhere else?

Reply:
I was originally going to use this for a test but in the end I didn't need it. I can put it back.